_includes/integrations/angular-tech-ref.md

```diff
@@ -275,6 +275,51 @@ For information setting the toolbar for {{site.productname}}, see: [User interfa
 ></editor>
 ```
 
+#### `modelEvents` (since v4.0.0)
+Used to specify the events that trigger the NgModelChange to emit. By default the wrapper uses `change input undo redo`.
+
+**Possible value:** A space separated list of TinyMCE editor events.
+
+**Type** String
+
+##### Example: `modelEvents`
+
+```html
+<editor
+  modelEvents="change input nodechange undo redo"
+></editor>
+```
+
+#### `allowedEvents` (since v4.2.0)
+Used to whitelist valid events to trigger from the editor to the Angular wrapper. By default, the wrapper will emit all the events in the [Event binding section]().
+
+**Possible values:** A comma separated list of events to allow.
```

Comment:
Not so much a docs issue but why does one setting split things by space and the other by commas.

Reply:
not sure, but also not a docs issue.

Reply:
Good point. Originally, it was because model events where not validated while the other events have to be bound into the component so they need to be parsed but it would make more sense if everything followed the same pattern